jfafard/struct1.html

```diff
@@ -12,19 +12,18 @@ <h1 class="c1">
                 Gallery
             </h1>
         </div>
-        <p align="center">
-            Click on either side of the picture to file between parts of the structure.
-        </p>
-        <p align="center">
-            <a href="struct1.jpg" target="_blank">Click here to see the full size image</a>
-        </p>
         <br>
-        <p align="center">
-            <img src="struct11.jpg" usemap="#backforth" alt="no alt text">
-        </p><map id="backforth">
-            <area alt="no alt" shape="rect" coords="0,0,350,525" href="struct8.html">
-            <area alt="no alt" shape="rect" coords="350,0,700,525" href="struct2.html">
-        </map>
+        <div class="marquee-outer">
+          <div class="marquee-inner">
+          <div class="row">
+            <div id="small-img">
+              <img src="structure1.jpg" class="img-responsive inline" alt="structure" height="3880px" width="16376px">
```

Comment:
The height and width attributes have one distinct difference from their CSS counterparts - they are always in pixels. There is no need for a px at the end. In fact, you need to _not_ have the px at the end. The purpose of this fix is to remove HTML errors - I'm not approving something that introduces more errors than it fixes. Remove the px from height and width